spec/features/bars/administrator_can_delete_bar_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+feature 'administrator user can delete a bar', %{
+  As an administrator
+  I want to delete an bar
+  So that no one can review it
+} do
+
+  # Acceptance Criteria
+  # [X] Administrator can delete bars from bar details page
+  # [X] Bar is no longer visible on index page
+  # [X] A user cannot delete a bar
+
+  scenario 'administrator can delete a bar' do
+    admin = FactoryGirl.create(:admin)
+    bar = FactoryGirl.create(:bar)
+
+    visit new_user_session_path
+
+    fill_in 'Email', with: admin.email
+    fill_in 'Password', with: admin.password
+
+    click_button 'Log in'
```

Comment:
At some point you might want to write a helper for this `def sign_in_as(user)` in your helpers folder.